utils/releaser/config/services.yaml

```diff
@@ -16,3 +16,7 @@ services:
 
     GuzzleHttp\Client: ~
     Symfony\Component\Console\Helper\QuestionHelper: ~
+
+    Shopsys\Releaser\FilesProvider\ComposerJsonFilesProvider:
+        arguments:
+            - ['packages', 'microservices', 'project-base']
```

Comment:
This should be set in the config: https://github.com/symplify/monorepobuilder#1-merge-local-composerjson-to-the-root-one

```yaml
# monorepo-builder.yml
parameters:
    package_directories:
        - 'packages'
        - 'microservices'
        - 'project-base' 
```

Reply:
well yes, however, it is not necessary here. I have created my own implementation - ` Shopsys\Releaser\FilesProvider\ComposerJsonFilesProvider` as I was not able to ignore `project-base/var` folder, see https://github.com/shopsys/shopsys/pull/623/commits/5b5a5ac7adab319c9efefdc8a19fdf82c26d758f :slightly_smiling_face: